vagrant_support/bashrc

```diff
@@ -0,0 +1,11 @@
+# .bashrc
+
+# Source global definitions
+if [ -f /etc/bashrc ]; then
+    . /etc/bashrc
+fi
+
+# User specific aliases and functions
+PS1='\[\e[0;31m\]\u\[\e[m\] \[\e[1;34m\]\w\[\e[m\] $ '
```

Comment:
This is not exactly a requirement, but is there to stylize the prompt on sshing into the vagrant box.